pkg/sources/gcs/gcs.go

```diff
@@ -52,7 +57,44 @@ type Source struct {
 	log        logr.Logger
 	chunksCh   chan *sources.Chunk
 
-	sources.Progress
+	processedObjects int32
```

Comment:
**question**: What does this do that `Progress.SectionsCompleted` doesn't?

Reply:
I think we can likely use `Progress.SectionsCompleted` here. I was using processedObjects when I had the initial implementation of progress tracking. We no longer need it. Nice rec, thx.